flexer/cli.py

```diff
@@ -72,6 +73,38 @@ def list(ctx):
         )
 
 
+@cli.command(name='new')
+@click.option('--name',
+              required=True,
+              help="A name for the new module")
+@click.option('--event-source',
+              required=True,
+              type=click.Choice(EVENT_SOURCES),
+              help="The event source for the module")
+@pass_context
+def new_module(ctx, name, event_source):
+    """
+    Create a new nFlex module.
+    """
+
+    module_type = ModuleTemplate.get_module_type(event_source)
+    click.echo(
+        'Creating a new {} module...'.format(module_type)
+    )
+
+    templates_dir = os.path.join(
+        os.path.dirname(__file__),
+        "templates",
+        module_type
+    )
```

Comment:
I thought the idea with Python was to ask for forgiveness rather than permission (i.e. catch the error if it comes up, rather than check if it exists beforehand). I don't mind checking first though, it's what I'm used to :)